experimental/QuadFormAndIsom/src/embeddings.jl

```diff
@@ -424,7 +424,7 @@ function _subgroups_orbit_representatives_and_stabilizers_elementary(Vinq::TorQu
   GVtoMGp = hom(GV, MGp, MGp.(act_GV); check = false)
   GtoMGp = compose(GtoGV, GVtoMGp)
 
-  @hassert :ZZLatWithIsom g-ngens(snf(abelian_group(H0))[1]) < dim(Qp)
+  g-ngens(snf(abelian_group(H0))[1]) >= dim(Qp) && return res
```

Comment:
Is your example small enough to be added as a test?

Reply:
Maybe yes, but I need to find it again. I will try to re-run my examples and find such example. (I fixed this on another machine last week and forgot to save the non-working example...)

Reply:
Found one, I will port it in a commit once I make sure the test goes well.